static.config.js

```diff
@@ -847,17 +873,23 @@ export default {
     }
 
     const routes = [
-      // {
-      //   path: '/search',
-      //   template: 'src/components/Pages/Search',
-      // },
       {
         path: '404',
         template: 'src/components/Pages/404',
       },
     ];
 
     const allLangs = Array.from(SUPPORTED_LANG_CODES);
+
+    // Adds languege url prefixes to static routes.
+    allLangs.forEach( lang => {
+      routes.push({
+        path: lang+'/search',
+        template: 'src/components/Pages/Search',
+        getData: () => getSearchIndex(createGraphQLClientsByLang(lang)),
```

Comment:
It looks like you're duplicating a query we're already making in makeAllPages(). It's the same allPagesQuery done a second time. Instead of fetching all of the site data a second time, could you use one call to Joplin to get the data for search and makeAllPages()? Your getSearchIndexQuery also isn't using any cursors which could be trying to transfer a huge amount of data in one chunk.

Reply:
👍You're right. I'm gonna take a stab at weaving `searchIndex` into `makeAllPages()`. Then, I should be able to remove `getData` from the `routes` object.